mod/gcforums/views/default/forms/gcforums/move_topic.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+
+$notice = "<p><form action='{$base_url}gcforums/move_topic'><div class='administrative-topic-tool'>
```

Comment:
nit: not sure the `<p>` should wrap the form

Reply:
do you know what the best uses for `<p></p>`? i've always thought `<br/>` was not the best thing to use for newlines

Reply:
I was looking at https://getbootstrap.com/docs/3.3/css/#forms, but inspecting the page I saw the the default for `<form>` is `display:block`. You could wrap it in a div, but I don't think it's necessary.